src/inspector/view/widgets/bendgridcanvas.cpp

```diff
@@ -59,6 +60,33 @@ static QPointF constrainToGrid(const QRectF& frameRectWithoutBorders, const QPoi
     return result;
 }
 
+QString pointAccessibleName(const CurvePoint& point)
+{
+    int fulls = point.pitch / 100;
+    int quarts = (point.pitch % 100) / 25;
+
+    QString string = QString::number(fulls);
+    if (fulls == 0 && quarts != 0) {
+        string = "";
+    }
+
+    switch (quarts) {
+    case 1:
```

Comment:
It would be better to add a static function and reuse it here and inside GuitarBend::computeBendText()